src/runner.py

```diff
@@ -536,6 +536,10 @@ def _configure_runner(self) -> None:
                 ["/usr/bin/chown", "-R", "ubuntu:ubuntu", "/home/ubuntu/.docker"]
             )
 
+        # Make python an alias of python3
+        self.instance.execute(["rm", "/usr/bin/python3"])
```

Comment:
Should this `rm` `/usr/bin/python` rather than `/usr/bin/python3`?

Reply:
Thanks this is fixed.